src/pcapi/serialization/decorator.py

```diff
@@ -88,10 +91,13 @@ def decorate_validation(route: Callable[..., Any]) -> Callable[[Any], Any]:
         if 403 not in on_error_statuses:
             on_error_statuses.append(403)
 
-        spectree_response = SpectreeResponse(*[f"HTTP_{on_error_status}" for on_error_status in on_error_statuses])
+        response_codes = (
+            set([f"HTTP_{on_success_status}"])
```

Comment:
(style / micro-détail)

```suggestion
            {f"HTTP_{on_success_status}"}
```